tests/goal_tests.cpp

```diff
@@ -143,7 +159,7 @@ TEST_CASE("pick_ik::make_pose_cost_fn") {
         auto const rotational_distance = 2.0;
         auto const rotation_scale = 0.5;
```

Comment:
Could you define `auto const position_scale = 1.0` here and use that throughout, for consistency with the rotation scale?